### ISSUE TITLE
[GANLatentDiscovery] Unsupervised Discovery of Interpretable Directions in the GAN Latent Space (Feb 2020 ICML)

### ISSUE THREAD

**Issue by andrewjong** [andrewjong/Deep-Learning-Paper-Surveys#15]
## 0. Article Information and Links

- Paper's project website: https://github.com/anvoynov/GANLatentDiscovery
- Release date: YYYY/MM/DD
- Number of citations (as of 2020/MM/DD): 

## 2. What's great compared to previous research?
- First unsupervised approach to discover semantic directions in latent spaces
- model-agnostic approach, no re-training required

- discover **practical**, interpretable, diverse, orthogonal directions
- example of immediate practical benefit of this for saliency detection.

- Can explore latent space in existing GAN models, and achieve similar style-based editing abilities as StyleGAN.

![image](https://user-images.githubusercontent.com/8121216/86195247-4a0fc700-bb05-11ea-90a7-c6f73077bb38.png)

## 3. Where are the key elements of the technology and method?

![image](https://user-images.githubusercontent.com/8121216/86197177-af65b700-bb09-11ea-940b-5a683b2b782e.png)

#### Two learnable components

1) Matrix `A`, size `d x K`, `d = latent dimension size`, `K = # interpretable directions`

2) Reconstruction Network `R` attempts to predict direction `k` and shift amount ` Ɛ`.

![image](https://user-images.githubusercontent.com/8121216/86197833-13d54600-bb0b-11ea-8e07-b9c4063e1e0a.png)

#### Loss function
![image](https://user-images.githubusercontent.com/8121216/86197629-a6291a00-bb0a-11ea-9457-ac6f718b5437.png)
Lcl is cross entropy, Lr is regression.

Encourage A to learn more meaningful directional changes that are easier for `R` to classify.

## 7. Are there any papers to read next?
[Related concurrent work: GANSpace](https://github.com/andrewjong/Deep-Learning-Paper-Surveys/issues/14)
